### PR TITLE
Setup prow CI 

### PR DESCRIPTION
**Pull request by kargakis**
Tested in https://github.com/kargakis/acs-engine/pull/115

### CODE REVIEW

**Comment by kargakis** [comment]
Tested in https://github.com/kargakis/acs-engine/pull/115
TODO:
- [ ] setup dns and update ingress resource

TLS can be fixed in a follow-up

File: .prowci/config.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+prowjob_namespace: default
+log_level: debug
+
+tide:
+  target_url: HOSTNAME/tide.html
```
[the remark this answers is not in the document]
Comment (CecileRobertMichon) [reply .prowci/config.yaml]: http://ci-bot-aks-ingress.eastus.cloudapp.azure.com/tide.html

**Comment by kargakis** [comment]
Waiting on https://github.com/kubernetes/test-infra/pull/8075

File: .prowci/config.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+prowjob_namespace: default
+log_level: debug
+
+tide:
+  # target_url: http://ci-bot-aks-ingress.eastus.cloudapp.azure.com/tide.html
```
Comment (CecileRobertMichon) [inline .prowci/config.yaml]: why is this commented out?

**Comment by kargakis** [comment]
Because of the issue in the ingress controller, the link is not working
properly atm.

**Comment by kargakis** [comment]
@CecileRobertMichon can you post logs from `hook`? I expected it to pick up this merge and update its config. You will need to do a `make prow-config-update` from `.prowci` and sync what we merged here with what's in the cluster.

**Pull request merged**